serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -87,13 +106,15 @@ The event of type `resume` is consumed by a link:{kogito_sw_examples_url}/server
     }
 ----
 
-This cloud event modifies the original value of the `move` field and publishes it as the `result` field. Moreover, it contains a Cloud Event attribute, named `kogitoprocrefid`, holding the process instance id of the workflow that published the consumed event.
+After that, the published CloudEvent modifies the value of `move` field and publishes the `move` field as a `result` field. Also, the published CloudEvent contains an attribute named `kogitoprocrefid`, which holds the process instance ID of the workflow that published the consumed CloudEvent. 
+
+The `kogitoprocrefid` attribute is crucial because when the correlation is not used, then this attribute is the only way for the Callback state to identify that the related CloudEvent needs to be used to resume the workflow.
```

Comment:
Can we highlight this paragraph somehow? This is really important.

Reply:
Lest compare
_Moreover, it contains a Cloud Event attribute, named `kogitoprocrefid`, holding the process instance id of the workflow that published the consumed event._
with 
_Also, the published CloudEvent contains an attribute named `kogitoprocrefid`, which holds the process instance ID of the workflow that published the consumed CloudEvent._ 
It = the consumed cloud event (consumer by the callback and published by the simulated external service) 
I agree the original sentence is ambigous, but the corrected one is plainly wrong, becuase it refers to the same cloud event as published and consumed

Reply:
Done, changed the sentence as follows:

Also, the consumed CloudEvent contains an attribute named `kogitoprocrefid`, which holds the process instance ID of the workflow that published the consumed CloudEvent.

Reply:
> Can we highlight this paragraph somehow? This is really important.

We can add a note to highlight the cruciality of kogitoprocrefid attribute.